build_tools/kokoro/gcp_ubuntu/cmake/riscv/riscv64-linux/build_kokoro.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
So to actually create a Kokoro build we need to add some config files internally. What I usually do when spinning up a new build is add those with everything disabled and the presubmit only manually triggered and then we can run the presubmit on the PR that introduces the config and iterate that way. Let me send you a CL adding the files.

(Just here for threading)

Reply:
Thanks for the help!